ocaml/xapi/xapi_pool_update.ml

```diff
@@ -265,7 +269,11 @@ let extract_update_info ~__context ~vdi ~verify =
        let url = attach_helper ~__context ~uuid:vdi_uuid ~vdi in
        let update_path = Printf.sprintf "%s/%s/vdi" Xapi_globs.host_update_dir vdi_uuid in
        debug "pool_update.extract_update_info get url='%s', will parse_file in '%s'" url update_path;
-       let xml = Xml.parse_file (String.concat "/" [update_path ; "update.xml"]) in
+       let xml = try
+         Xml.parse_file (String.concat "/" [update_path ; "update.xml"])
```

Comment:
Use `Filename.concat update_path "update.xml"`. I understand that the code was already there but this is an opportunity to fix it.